src/instructlab/log.py

```diff
@@ -37,17 +38,59 @@ def isatty(self):
         return False
 
 
-def stdout_stderr_to_logger(logger, log_file):
-    if log_file:
-        # Use the existing log file if it exists and append to it
-        mode = "a"
-        # Create file handler
-        file_handler = logging.FileHandler(log_file, mode, encoding="utf-8")
-        logger.addHandler(file_handler)
-        # Create formatter and set it for both handlers
-        formatter = CustomFormatter(FORMAT)
-        file_handler.setFormatter(formatter)
+def write_log_to_file(logger: logging.Logger, log_file: pathlib.Path | None) -> None:
```

Comment:
Wondering whether there is a better name for this?  Suggestions: `add_file_handler` or `setup_file_logger`

Reply:
Renamed to `add_file_handler_to_logger`. Thanks